gnocchi/indexer/sqlalchemy.py

```diff
@@ -1131,6 +1131,13 @@ def _build_sort_keys(sorts, unique_keys):
         return sort_keys, sort_dirs
 
 
+def _operator_in(field_name, value):
+    # Do not generate empty IN comparison
+    # https://github.com/gnocchixyz/gnocchi/issues/530
+    if len(value):
+        return field_name.in_(value)
```

Comment:
This looks wrong to me. If I do "{in: {id: []}}, I expect nothing to be returned. With this change the whole table will be returned, no ?

Reply:
This returns `None` which is equivalent to filter on `False`, i.e., nothing matches.
(that's my understanding since it works :-p)